pkg/cache/cache.go

```diff
@@ -721,6 +725,20 @@ func (c *Cache) MatchingClusterQueues(nsLabels map[string]string) sets.Set[strin
 	return cqs
 }
 
+// ShouldCheckWorkloadsPreemption returns true if it has workloads pending preemption now.
+func (c *Cache) ShouldCheckWorkloadsPreemption() bool {
```

Comment:
```suggestion
func (c *Cache) HasPreemptingWorkloads() bool {
```

Reply:
The main issue with this naming is that depending on the policy, we have workloads needing preemption now and needing preemption later.

Reply:
ok, but the name should reflect what this function calculates, not what this is used for.